SFTP/connector.json

```diff
@@ -0,0 +1,54 @@
+{
+	"name": "SFTP",
+	"fileName": "connector.js",
+	"fields": [
+		{
+			"type": "String",
+			"key": "host",
+			"label": "Host",
+			"htmlInputType": "text",
+			"encrypted": false,
+			"required": true
+		},
+		{
+			"type": "Integer",
+			"key": "port",
+			"label": "Port",
+			"htmlInputType": "text",
+			"encrypted": false,
+			"required": true
+		},
+		{
+			"type": "String",
+			"key": "user",
+			"label": "User",
+			"htmlInputType": "text",
+			"encrypted": false,
+			"required": true
+		},
+		{
+			"type": "String",
```

Comment:
This is not a required field.

Reply:
User and password are always going to be required, aren't they?